tests/post_training/test_quantize_conformance.py

```diff
@@ -271,7 +283,12 @@ def test_weight_compression(
         pipeline_cls = test_model_param["pipeline_cls"]
         pipeline_kwargs = create_pipeline_kwargs(test_model_param, subset_size, test_case_name, wc_reference_data)
         pipeline_kwargs.update(
-            {"output_dir": output_dir, "data_dir": data_dir, "no_eval": no_eval, "run_benchmark_app": run_benchmark_app}
+            {
+                "output_dir": output_dir,
+                "data_dir": data_dir,
+                "no_eval": no_eval,
+                "run_benchmark_app": run_benchmark_app,
```

Comment:
please add "batch_size": batch_size for consistency.

Reply:
Done